tests/integration/go.mod

```diff
@@ -0,0 +1,66 @@
+module github.com/kyma-incubator/api-gateway/tests/integration
```

Comment:
I wanted to have it in a submodule and not mix it up with the api-gateway controller deliverable. Are you sure you want to have everything in one module ?

Reply:
I think that's the recommendation for Go modules so it might be the expected configuration for some tools, but if you feel that would be better since we only use it for tests it can stay

Reply:
I think I like it to be separate. Will see how it looks in a single module and adapt it if it looks better.